corehq/apps/cloudcare/templates/cloudcare/preview_app_base.html

```diff
@@ -51,13 +51,14 @@
   <script src="{% statici18n LANGUAGE_CODE %}"></script> {# DO NOT COMPRESS #}
 
   {% include "hqwebapp/partials/requirejs.html" %}
+  {% include "hqwebapp/partials/webpack.html" %}
```

Comment:
Is it fine to be including both of these? Seems like it should be one or the other.

Reply:
It's not harmful, because all the logic in `requirejs.html` checks for a `requirejs_main` value, but yes it can be removed, I just missed it. ff214d8195e